cluster/addons/device-plugins/nvidia-gpu/daemonset.yaml

```diff
@@ -35,9 +35,15 @@ spec:
       - name: dev
         hostPath:
           path: /dev
+      - name: varlog
+        hostPath:
+          path: /var/log
       containers:
       - image: "gcr.io/google-containers/nvidia-gpu-device-plugin@sha256:0e79da6998a61257585e0d3fb5848240129f0fa5b4ad972dfed4049448093c33"
-        command: ["/usr/bin/nvidia-gpu-device-plugin", "-logtostderr"]
```

Comment:
Would I still be able to use `kubectl logs` to see the logs?

Also, why is this needed at all? Doesn't logging to stderr automatically go to stackdriver etc.?